runtimes/pythonrt/py_test.go

```diff
@@ -78,3 +78,34 @@ func processEnv(t *testing.T, pid int) map[string]string {
 	}
 	return env
 }
+
+func genExe(t *testing.T, name string) {
+	file, err := os.Create(name)
+	require.NoError(t, err)
+	file.Close()
+	err = os.Chmod(name, 0o766)
```

Comment:
why do you need to chmod?

Reply:
`LookPath` checks that the file is executable.

Reply:
this will run in tests, but may fail on actual system
```shell
lrwxrwxrwx - root 12 Feb 17:50  /usr/bin/python3 -> python3.11
```
python may be a link , thus with 777 mod

Reply:
It will work with a link since the link is an executable.